doc/user_manual/rom.tex

```diff
@@ -3024,12 +3034,9 @@ \subsubsection{ARMA}
     two distinct series: one containing times in which the original target is zero, and one in the remaining
     times. The results from separated ARMAs are recombined after sampling. This can be a methodology for
     treating histories with long zero-value segments punctuated periodically by peaks.
-  \item \xmlNode{Fourier}, \xmlDesc{comma-separated integers, optional field}, must be positive integers. This defines the
-    based period that will be used for Fourier detrending, i.e., this field defines $1/f_m$ in the above equation.
-    When this filed is not specified, the ARMA considers no Fourier detrend.
-  \item \xmlNode{FourierOrder}, \xmlDesc{comma-separated integers, optional field}, must be positive integers. The number of integers specified
-    in this field should be exactly same as the number of base periods specified in the node \xmlNode{Fourier}.
-    This field defines $K_m$ in the above equation.
+  \item \xmlNode{reseedCopies}, \xmlDesc{boolean, optional field}, if True then whenever the ARMA is copied, a
+    random reseeding will be performed to ensure different histories.
+  \default{True}
```

Comment:
I did not see the changes regarding this node in this PR.

Reply:
There were no changes to this feature in this request.